swagger.yaml

```diff
@@ -7002,9 +7028,11 @@ definitions:
         items:
           type: string
         description: List of CPU architectures provided by the image.
+        x-go-custom-tag: gorm:"type:text[]"
```

Comment:
does it require any DB migration? what this means?
question holds for any `gorm` option being added/changed

Reply:
This one is a left over from when I considered saving the CPU architectures in the DB, but it is not relevant anymore, thanks

Reply:
As for the `primaryKey`, I am not sure what kind of migration do you mean? this is a new table so it will apply from the get-go